src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -131,7 +131,7 @@ public void identify(IdentifyPayload identify) {
   public void screen(ScreenPayload screen) {
     super.screen(screen);
     if (trackAllPages) {
-      event(String.format(VIEWED_EVENT_FORMAT, screen.event()), screen.properties(), null);
+      event("Viewed Screen", screen.properties(), null);
```

Comment:
Not sure if this was talked about, but this is a breaking change. IMO we should have this be a new setting.